packages/devel/dbus-glib/package.mk

```diff
@@ -17,7 +17,7 @@
 ################################################################################
 
 PKG_NAME="dbus-glib"
-PKG_VERSION="0.104"
+PKG_VERSION="0.106"
```

Comment:
this also requires a URL update, see https://github.com/OpenELEC/OpenELEC.tv/pull/4687